documentation/2.0/content/userguide/target_env.md

```diff
@@ -42,9 +42,11 @@ These target environment configurations are included in the WebLogic Deploy Tool
 
 You can use these targets to customize the model and create a domain resource file for use with WebLogic Kubernetes Operator. There are three targets for specific [domain home source types](https://oracle.github.io/weblogic-kubernetes-operator/userguide/managing-domains/choosing-a-model/):
 
-- `wko` for [Model in Image](https://oracle.github.io/weblogic-kubernetes-operator/userguide/managing-domains/model-in-image/) deployments
-- `wko-dii` for Domain in Image deployments
-- `wko-pv` for Domain in PV deployments
+- `wko` and `wko4` for [Model in Image](https://oracle.github.io/weblogic-kubernetes-operator/userguide/managing-domains/model-in-image/) deployments
```

Comment:
In WKO 4.0, the link to the Model in Image docs has been changed to https://oracle.github.io/weblogic-kubernetes-operator/managing-domains/model-in-image/

Reply:
Updated, including changes for broken links elsewhere.